sphinx/quickstart.py

```diff
@@ -471,12 +471,12 @@
 
 # User-friendly check for sphinx-build
 ifeq ($(shell which $(SPHINXBUILD) >/dev/null 2>&1; echo $$?), 1)
-$(error \
+\t$(error \
 The '$(SPHINXBUILD)' command was not found. Make sure you have Sphinx \
 installed, then set the SPHINXBUILD environment variable to point \
 to the full path of the '$(SPHINXBUILD)' executable. Alternatively you \
 can add the directory with the executable to your PATH. \
-If you don't have Sphinx installed, grab it from http://sphinx-doc.org/)
+If you don\'t have Sphinx installed, grab it from http://sphinx-doc.org/)
```

Comment:
This will result in the same string, since Python will interpret the backslash-quote as a quote...